server/src/main/java/org/opensearch/transport/TransportService.java

```diff
@@ -906,6 +923,54 @@ public String toString() {
         }
     }
 
+    private String createSpanName(String action, Transport.Connection connection) {
```

Comment:
Another one `SpanBuilder.from(String action, Transport.Connection connection)`?

Reply:
This is not being used, will remove it.